### PR TITLE
Torsten/cache rev

### PR DESCRIPTION
**Pull request by torsten-deriv**

### CODE REVIEW

File: lib/App/Config/Chronicle.pm [hunk 1]
```diff
@@ -366,13 +366,24 @@ sub _save_dynamic {
...
 
 =cut
-
-sub current_revision {
-    my $self = shift;
-    my $settings = $self->chronicle_reader->get($self->setting_namespace, $self->setting_name);
-    return $settings->{_rev};
+{
+    my %c;
+    sub current_revision {
```
Comment (michaelmueller-binary) [inline lib/App/Config/Chronicle.pm:373]: please use proper variable names.

File: lib/App/Config/Chronicle.pm [hunk 2]
```diff
@@ -366,13 +366,24 @@ sub _save_dynamic {
...
+        my $self = shift;
+        my $t = int(time/2);
```
Comment (michaelmueller-binary) [inline lib/App/Config/Chronicle.pm:375]: Time is better declared as a clearly named constant

File: lib/App/Config/Chronicle.pm [hunk 3]
```diff
@@ -366,13 +366,24 @@ sub _save_dynamic {
...
+        my $k = $self->setting_name . "\0" . $self->setting_namespace;
+        my $el = $c{$k};
+        return $el->[1] if $el and $el->[0]>=$t;
```
Comment (billmarriott-deriv) [inline lib/App/Config/Chronicle.pm:378]: Agree with @michaelmueller-binary, these variables are quite cryptic :)

**Comment by tom-binary** [comment]
Since this is no longer active, I'm closing the PR.

**Pull request closed without merging**